decidim-proposals/app/views/decidim/proposals/proposals/_edit_form_fields.html.erb

```diff
@@ -3,7 +3,7 @@
 </div>
 
 <div class="field hashtags__container">
-  <%= form.text_area :body, rows: 10, class: "js-hashtags", hashtaggable: true, value: form_presenter.body(extras: false).strip %>
+  <%= form.text_area :body, rows: 10, class: "js-hashtags js-mentions", hashtaggable: true, value: form_presenter.body(extras: false).strip %>
```

Comment:
I think that mentions can not be done in the body of a proposal... do you really need to add this css class?

Reply:
removed